WcaOnRails/app/views/competitions/index.html.erb

```diff
@@ -87,12 +89,11 @@ $(".js-scroll-to").click(function(e) {
   if(!$(this).hasClass('disabled')) {
     // Little hack to activate the affix thing and
     // move correctly to the anchor.
-    $("#simple-nav").removeClass('affix-top').addClass('affix');
```

Comment:
Well, it behaves as expected here (no page jump when scrolling, and clicking goes to the right position.)